src/assets/stylesheets/_price_calc.scss

```diff
@@ -257,7 +262,7 @@ $color-lighter-gray: #d8d6d6;
 }
 .price-calc__disk {
   .price-calc__included {
-    width: calc(45.5% - 6px);
+    width: calc(71.5% - 6px);
```

Comment:
The TB disk mark is way out now for what's included in the PHI-ready plan. Most DBs running on aptible are in the tens of GBs.